Base/usr/share/man/man1/Magnifier.md

```diff
@@ -0,0 +1,21 @@
+## Name
+
+![Icon](/res/icons/16x16/app-magnifier.png) Magnifier - Magnifier application
+
+[Open](file:///bin/Magnifier)
+
+## Synopsis
+
+```**sh
+$ Magnifier
+```
+
+## Description
+
+`Magnifier` is an application that magnifies or zooms-in an area of the desktop where the mouse cursor hovers.
+
+You can launch _Magnifier_ by selecting Serenity menu -> Utilities -> Magnifier.
+
+Magnifier can do 2x, 4x, or 8x zoom in realtime. Once you feel that the area being captured is just right, you can pause capture by pressing the shortcut key Spacebar.
+
+For users with sight impairment, it has the facility to apply several filters on the captured image. Click on _Accessibility_ menu and select one of the impairment category available on the drop-down list.
```

Comment:
s/category/categories/
s/on _Accessibility_ menu/on the _Accessbility menu/ perhaps?

Reply:
Recommendation applied on singular->plural form of category, Accessibility retained as per https://dict.org/bin/Dict?Form=Dict2&Database=*&Query=accessibility

Reply:
I agree with demostanis.